plaidml/op/op.h

```diff
@@ -599,10 +599,35 @@ inline edsl::Tensor reorg_yolo(const edsl::Tensor& I, int stride, bool decrease,
   return details::op("reorg_yolo", args).as_tensor();
 }
 
-inline edsl::Tensor repeat(const edsl::Tensor& I, int repeats, int axis) {
-  auto args = edsl::make_tuple(I, repeats, axis);
-  return details::op("repeat", args).as_tensor();
-}
+class repeat {
+ public:
+  explicit repeat(const edsl::Tensor& I) : I_(I) {}
+
+  repeat& repeats(int repeats) {
```

Comment:
Maybe call this `count` to avoid confusion?

Reply:
Sure. The `repeats` matches numpy, but I agree that `count` is a less confusing name, so I made the switch.